incubator/command-generate/src/test/resources/io/aklivity/zilla/runtime/command/generate/internal/openapi/http/proxy/jwt/openapi.yaml

```diff
@@ -39,15 +39,6 @@ paths:
       responses:
         '200':
           description: A paged array of items
-          headers:
-            x-next:
-              description: A link to the next page of responses
-              schema:
-                type: string
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Items"
```

Comment:
We need to generate config to validate OpenAPI response headers and content schemas, agree?

Reply:
We generate config which includes validation in the `validator` and the `complete` test cases, and config which doesn't include validation in the `plain`, `tls` and `jwt` test cases.